apps/laboratory/tests/shared/validators/ModalValidator.ts

```diff
@@ -9,6 +9,10 @@ export class ModalValidator {
     await expect(this.page.getByTestId('account-button')).toBeVisible()
   }
 
+  async expectAuthenticated() {
+    await expect(this.page.getByTestId('w3m-authentication-status')).toContainText('authenticated')
```

Comment:
Blocker: `unauthenticated` contains `authenticated`. You may want to include the `Status:` prefix